### DISCUSSION TITLE
`elfutils` fails to build from source

### DISCUSSION THREAD

**Discussion by jacktose** [Homebrew/discussions#4840] [Linux]
### Additional Info
I am _not_ using Windows Subsystem for Linux.

### Output of `brew config`
https://gist.github.com/jacktose/a8c67ca3535ad72ac5c1f35148b0ef17#file-00-config-out

### Output of `brew doctor`
https://gist.github.com/jacktose/a8c67ca3535ad72ac5c1f35148b0ef17#file-00-doctor-out
(Yes, it's messy. This is on a system I don't control, which is why I'm using brew in the first place.)

### Description of issue

Installation of `elfutils` (from source) always fails. (It's a dependency: `vim`→`ruby`→`rust`→`elfutils`.)

[`brew gist-log` here.][gist] [The key error](https://gist.github.com/jacktose/a8c67ca3535ad72ac5c1f35148b0ef17#file-config-log-L665) seems to be:
```
configure:8804: error: __cxa_demangle not found in libstdc++, use --disable-demangler to disable demangler support.
```

This is a linuxbrew installation without sudo, in a non-standard prefix. I know it's not supported, but I'd appreciate any help pointing me to the root of this issue or a fix. I'm not great with compilers.

[gist]: https://gist.github.com/jacktose/a8c67ca3535ad72ac5c1f35148b0ef17

**Comment by bangbangbear** [comment]
I got the same issue here, and my work around is to use "brew install --force-bottle elfutils" to install binaries instead of building from the source. I think the solution is to add "--disable-demangler" to the config file elfutils.rb, but I am not able to do it.
